src/base/porting.h

```diff
@@ -349,11 +349,7 @@ extern int free_space (const char *, int);
 #define SETJMP _setjmp
 #endif
 
-#if defined (WINDOWS)
 #define GETHOSTNAME(p, l) css_gethostname(p, l)
```

Comment:
TODO as separate task - unify tpc/wintpc; use css_gethostname directly instead of GETHOSTNAME; remove GETHOSTNAME from porting.h

http://jira.cubrid.org/browse/CBRD-22765 may be used for this purpose.

Reply:
please find http://jira.cubrid.org/browse/CBRD-22861 for TODOs :)